src/main/java/Onandon.java

```diff
@@ -0,0 +1,211 @@
+        String description;
+        String by;
+        int indexBy;
+
+        indexBy = inputText.indexOf("/by");
+        description = inputText.substring(9, indexBy-1);
+        by = inputText.substring(indexBy + 4);
+
+        Print.printFormAbove();
+        tasks.add(new Deadline(description, by));
+        System.out.println("\t\t" + tasks.get(cnt).toString());
+        cnt += 1;
+        Print.printCnt(cnt);
+
+        return cnt;
+    }
+
+    public static int printEventAction(String inputText, ArrayList<Task> tasks, int cnt){
+        String description;
+        String from;
+        String to;
+        int indexFrom;
+        int indexTo;
+
+        indexFrom = inputText.indexOf("/from");
+        indexTo = inputText.indexOf("/to");
+        description = inputText.substring(6, indexFrom-1);
+        from = inputText.substring(indexFrom+6, indexTo-1);
+        to = inputText.substring(indexTo+4);
+
+        Print.printFormAbove();
+        tasks.add(new Event(description, from, to));
+        System.out.println("\t\t" + tasks.get(cnt).toString());
+        cnt += 1;
+        Print.printCnt(cnt);
+
+        return cnt;
+    }
+
+    public static void printMarkAction(String inputText, ArrayList<Task> tasks){
+        int index;
+
+        Print.printMark();
+        index = Integer.parseInt(inputText.split(" ")[1])-1;
+        tasks.get(index).markAsDone();
+        System.out.println("\t\t[" + tasks.get(index).getStatusIcon() + "] " +
+                tasks.get(index).getDescription());
+        Print.printUnderline();
+    }
+
+    public static void printUnmarkAction(String inputText, ArrayList<Task> tasks){
+        int index;
+
+        Print.printUnmark();
+        index = Integer.parseInt(inputText.split(" ")[1])-1;
+        tasks.get(index).markAsUnDone();
+        System.out.println("\t\t[" + tasks.get(index).getStatusIcon() + "] " +
+                tasks.get(index).getDescription());
+        Print.printUnderline();
+    }
+
+    public static void printListOfTasks(ArrayList<Task> tasks, int cnt){
+        Print.printList();
+        for(int i=0; i<cnt; i++){
+            System.out.println("\t " + (i+1) + ". " + tasks.get(i).toString());
+        }
+        Print.printUnderline();
+    }
+
+    public static int printDeleteOfTasks(String inputText, ArrayList<Task> tasks, int cnt){
+        int index;
+
+        Print.printDelete();
+        index = Integer.parseInt(inputText.split(" ")[1])-1;
+        System.out.println("\t\t" + tasks.get(index).toString());
+        tasks.remove(index);
+        cnt -= 1;
+        System.out.println("\t Now you have " + cnt + " tasks in the list.");
+        Print.printUnderline();
+        return cnt;
+    }
+
+    public static void checkException(String inputText) throws OnandonEmptyException, OnandonUnknownException, OnandonNotaskException {
+        String[] split = inputText.split(" ");
+        String tgt = inputText.split(" ")[0];
```

Comment:
Try to give better names to your variables to improve readability and for better understanding of the code.